### PR TITLE
add failed and succesful file in getsharedfile

### PR DESCRIPTION
**Pull request by nettashviro**

### CODE REVIEW

**Review by Shahar-Y — APPROVED** [review]
We neet to check with Rabas that they won't be affected due to these changes. Do not push to develop yet

**Comment by nettashviro** [comment]
> We neet to check with Rabas that they won't be affected due to these changes. Do not push to develop yet

the changes only affected getSharedFIles... rabaz not supposed to call this function

File: file/file.go [hunk 1]
```diff
@@ -521,17 +528,24 @@ func (r *Router) GetSharedFiles(c *gin.Context, queryAppID string) {
...
-			files = append(
-				files,
+			filesSuccesful = append(
+				filesSuccesful,
 				CreateGetFileResponse(file, permission.GetRole().String(), userPermission),
 			)
 		}
 	}
 
+	var errMsg string
```
Comment (yoavrozner) [inline file/file.go:538]: I think it should return a semi-failed response code such as 202.
[https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202](url)

**Review by yoavrozner — COMMENTED** [review]
Well done @nettashviro !

**Review by Shahar-Y — APPROVED** [review]
Good job. Notice the conflict in your favorite file :)

**Review by Shahar-Y — APPROVED** [review]

**Comment by nettashviro** [comment]
@Shahar-Y

**Pull request merged**